### DISCUSSION TITLE
Github Desktop

### DISCUSSION THREAD

**Discussion by Khanyisa-Ngejane** [community/community#56221] [Enterprise]
### Select Topic Area

Question

### Body

"Unable to authenticate with the Github Enterprise instance. Verify that the URL is correct, that your Github Enterprise instance is running version 3.0.0 or later, that you have an internet connection and try again"

Hi all, got this massage while trying to authenticate to github desktop app. 
I have tried the solutions provided on the desktop repo but not working, I also tried the stack overflow. No luck

**Comment by marknn3** [comment]
Use the GitHub.com Sign-in, even though you have an Enterprise account. This worked for me:
![image](https://github.com/community/community/assets/4366965/5d1f5750-9476-418b-8c70-7eead9012673)
